plugins/modules/azure_rm_roleassignment_info.py

```diff
@@ -22,29 +23,44 @@
     - Gets facts of Azure Role Assignment.
 
 options:
-    scope:
+    assignee:
```

Comment:
It is recommended that the two related modules have the same parameters, thank you!
```suggestion
   assignee_object_id：
```

Reply:
I agree with your proposed design change, but I did not add or change the assignee parameter, it was only a re-ordering of the documentation to match the order that it's displayed (alphabetical sorting.) I am in favor of assignee_object_id also because it is explicitly the guid and not compatible with, for example, an assignee name, and also matches the naming for the az cli parameter flag.

Would it be an acceptable solution to make the parameter "assignee_object_id" with an alias of "assignee" so that we can retain backwards compatibility with existing playbooks? I would not want to inadvertently break existing playbooks.